CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Changelog
 
 Release: dd.mm.yyyy
 
+### New
+
+- Add "Edit configuration" status menu item.
```

Comment:
One last request, use pretty quotes i.e. “Edit configuration” and add a reference to this PR to the end of the line.

I’m pedantic, I know. :smiley: Also, squash the commits if you can. Thanks!